lisa/energy_meter.py

```diff
@@ -450,6 +453,14 @@ def report(self, out_dir, out_energy='energy.json'):
         logger = self.get_logger()
         channels_nrg = {}
         channels_stats = {}
+
+        # iio-capture returns and empty string if killed right after its
+        # invocation, so to workaround the issue, make sure reset was long ago
+        # enough
+        delta = time.monotonic() - self.reset_time
+        if delta < 2.0:
```

Comment:
I know I said we'll ditch this and move on to use devlib's ACME, but... Can we use a class attribute instead?
Something like this:

```python
class ACME(EnergyMeter):
    # ...
    REPORT_DELAY_S = 2.0
    """
    iio-capture returns an empty string if killed right after its invocation,
    so we have to enforce a delay between reset() and report()
    """

    # ...

    delta = time.monotonic() - self.reset_time
    if delta < self.REPORT_DELAY_S:
        sleep(self.REPORT_DELAY_S - delta)
```

Reply:
Sure